quna/src/main/java/com/ktpt/quna/domain/model/QuestionQueryDao.java

```diff
@@ -0,0 +1,27 @@
+package com.ktpt.quna.domain.model;
+
+import static com.ktpt.quna.domain.model.QMember.*;
+import static com.ktpt.quna.domain.model.QQuestion.*;
+
+import java.util.List;
+
+import org.springframework.stereotype.Repository;
+
+import com.querydsl.jpa.impl.JPAQueryFactory;
+
+@Repository
+public class QuestionQueryDao {
```

Comment:
DAO는 DataAccessObject의 약자이니 대문자를 쓰는건 어떤가요?
```suggestion
public class QuestionQueryDAO {
```